### PR TITLE
Set versions workflow

### PR DESCRIPTION
**Pull request by vietj**

### CODE REVIEW

File: .github/workflows/set-versions.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      stableSnapshotVersion:
+        description: 'Vert.x stable snapshot version, e.g. 1.2.4-SNAPSHOT'
+        required: true
+        type: string
+jobs:
+  set-versions:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
```
Comment (vietj) [inline .github/workflows/set-versions.yml:18]: @tsegismont removed the branch name there so it should use the current branch the workflow is applied to

**Comment by vietj** [comment]
Did a mistake with branches, so I created a new one @tsegismont

**Review by tsegismont — APPROVED** [review]
LGTM, thank you @vietj !

**Comment by tsegismont** [comment]
@vietj anything else you want to update before merging?

**Pull request merged**